plugins/@grouparoo/google-sheets/src/lib/sheet-import/downloadAndRefreshFile.ts

```diff
@@ -0,0 +1,63 @@
+import { log } from "actionhero";
+import fs from "fs-extra";
+import path from "path";
+import { tmpdir } from "os";
+import { SimpleSourceOptions } from "@grouparoo/core";
+import Spreadsheet from "./Spreadsheet";
+import csvWriter from "csv-write-stream";
+
+const CSV_CACHE_MILLISECONDS = 1000 * 60;
+
+export async function downloadAndRefreshFile(
```

Comment:
Can this method be shared with the CSV plugin somehow?

Reply:
I spent a good while thinking about it but nothing better came out. Do you have suggestions for it?

Reply:
It could be possible with a combination API that looks like `downloadAndRefreshFile(cacheTime:number, fileName:string, reader:function)`, but it's probably not worth it at the moment - this is OK!